addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -200,6 +200,8 @@ export class Chatbot extends Record {
         );
         if (!redirectionAlreadyDone) {
             browser.location.assign(answer.redirect_link);
+        } else if (this.store.env.services.ui.isSmall) {
+            this.store.chatHub.opened[0]?.fold();
```

Comment:
Can't we do it before navigating? I feel like it would be clearer than the actual fix: while it works, it relies too much on implementation details (i.e. the fact that the chat bot state is played back when loading the new page). Also, we could add a tour to cover this scenario.

Reply:
Folding before can be confusing: the opening of other page takes time and we can wonder why it's being folded.
But at the same time, it will probably show more consistently the unread counter than now.

The best between both easiest fix is most likely the fold before, the unread counter on the folded chat bubble is a more important concern than the early folding and page loading.

IMO the late folding like this PR and the reliable showing of unread counter on chat bubble would be the ideal solution.